content/v2.0/query-data/optimize-queries.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Optimize Flux queries
+description: >
+  Optimize your Flux queries by changing function order, understanding how functions work,
+  and being conscious of the nature of the data queried.
+weight: 104
+menu:
+  v2_0:
+    name: Optimize queries
+    parent: Query data
+v2.0/tags: [query]
+---
+
+Optimize your Flux queries with a few simple principles.
+Query optimizations center around reducing the memory and compute (CPU) requirements
+of a query by changing function order, understanding how functions work, and being
+conscious of the nature of the data queried.
```

Comment:
Optimize your Flux queries by reducing the memory and compute (CPU) that your queries require:
- Change the function order
- Understand how functions work (not sure this one explicitly reduces memory and compute?)
- Consider the nature of queried data (not sure what "nature" means here)